ood-portal-generator/share/ood_portal_example.yml

```diff
@@ -379,6 +379,18 @@
 # Default: {} (empty hash)
 #oidc_settings: {}
 
+# The Dex URI behind Apache reverse proxy
+# Setting this value to some path will result in Dex listening on localhost
+# as well as only using HTTP for proxied communication
+# Example:
+#   dex_uri: /dex
+# Default: null
+#dex_uri: null
+
+# The Dex HTTP port.
+# This is considered a private value that is set by the dex.http_port below
+#dex_http_port: 5556
```

Comment:
I take it you added this here because tests fail? You can add it to this blocklist and remove it and the comment if that's the case.  

https://github.com/OSC/ondemand/blob/b7538dd6ec697dea4b203e5227329344c78b53f4/ood-portal-generator/spec/ood_portal_generator_view_spec.rb#L18

These tests are just to be sure we update this example file with meaningful stuff like `dex_uri`.

Reply:
Yep, I've removed from example config and excluded this view item from that test.